_data/simple-icons.json

```
@@ -8846,6 +8846,16 @@
             "hex": "1C1B1C",
             "source": "https://www.peakdesign.com"
         },
+        {
+            "title": "Pearson",
+            "hex": "000000",
+            "source": "https://www.pearson.com",
+            "aliases": {
+                "aka": [
+                    "Pearson VUE"
+                ]
```

I know I suggested the `aka` of Pearson VUE, but this is probably a better way to handle it, given their hex codes are different...

```suggestion
                "dup": [
                    {
                        "title": "Pearson VUE",
                        "hex": "008FB4"
                    }
                ]
```